src/OpenTelemetry.AutoInstrumentation.Logging/CustomLogger.cs

```diff
@@ -14,7 +14,7 @@
 // limitations under the License.
 // </copyright>
 
-using System.Runtime.CompilerServices;
+using System;
```

Comment:
I think we can rename `CustomLogger` to `Logger`. Then we could move the `OtelLogging` code here and rename `GetLogger()` to `Get()` or even `Value`.

This shoud be addressed in a separate PR.

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/2022